Source/cmGlobalFastbuildGenerator.cxx

```diff
@@ -2401,6 +2401,17 @@ class cmGlobalFastbuildGenerator::Detail::Generation
 
 					linkPath = frameworkPath + linkPath;
 
+					if(target.IsExecutableWithExports())
+					{
+						const char* defFileFlag = lg->GetMakefile()->GetDefinition("CMAKE_LINK_DEF_FILE_FLAG");
+						const std::string config = lg->GetMakefile()->GetSafeDefinition("CMAKE_BUILD_TYPE");
```

Comment:
It sure does.

I can't run the tests right now but before the modification this fixed 2 tests.